SpaceInvaders/Aliens/Strategies/ShootUsingRandomStrategy.cs

```diff
@@ -64,7 +64,6 @@ public class RandomStrategySelector : IBinaryStrategySelector
     {
         private readonly int _chancesForFirstStrategy;
         private readonly int _chancesForSecondStrategy;
-        private readonly Random _rng = new Random();
```

Comment:
@leppie,
Simply making both classes static has an undesirable effect on shooting behavior. 1/3 of our alien population will not ever be picked to shoot.

Both Randoms still seed to the same value. Both Randoms will generate the exact sequence of numbers. StrategySelector executes, selects the RandomShootingStrategy when random number is larger than 0.
The Random in RandomShootingStrategy uses the same seed. It's random number will also be larger than 0.
This means our first alien will never be picked to shoot. 
Only 2/3 of our alien population will have chance to shoot, leaving 1/3 never shooting.